rb_ws/src/buggy/scripts/auton/model_predictive_controller.py

```diff
@@ -44,28 +47,25 @@ class ModelPredictiveController(Controller):
     N_CONTROLS = 1
 
     # MPC Cost Weights
-    state_cost = np.array([0.0001, 250, 5, 25])  # x, y, theta, steer
+    state_cost = np.array([0.0001, 250, 5, 2500])  # x, y, theta, steer
```

Comment:
Why did we increase state cost for steer by 100x?

Reply:
MPC too agressive.

Reply:
this was the params we used during LGAP last week